src/main/java/com/cburch/logisim/fpga/download/Download.java

```diff
@@ -89,6 +88,7 @@ public Download(
       boolean writeToFlash,
       boolean downloadOnly,
       boolean generateHdlOnly) {
+    this.progressBar = null;
```

Comment:
Why not have `null` assigned directly in line 43?

Reply:
Good point.